live/root/usr/bin/agama-dud

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# Experimental DUD funtionality for Agama
+#
+# Usage: inst.dud=URI [inst.dud=URI]
+#
+# This is a very simple version of the DUD functionality
+# - There can be multiple inst.dud=URI entries, but...
+# - All of them are considered to be RPMs
+# - The script does not check if downloading succeeded
+# - Or for any deps of the package
+# - Of whether installation/upgrade succeeded
+# - Does not restart itself if the script is updated
+
+# check if there is any DUD
+if ! grep -q "\b\(inst\|agama\)\.dud=.\+\b" /run/agama/cmdline.d/agama.conf; then
+  echo "No DUD present"
+  exit 0
+fi
+
+# Make sure that it does not refresh
+zypper modifyrepo --disable --all
+
+for URI in `sed -n 's/\(.*[[:space:]]\|^\)\(inst\|agama\)\.dud=\([^[:space:]]\+\).*/\3/p' /run/agama/cmdline.d/agama.conf`; do
```

Comment:
in a PBI about renaming install parameters we discussed that backward compatibility with "old" ```agama.``` prefix should be just temporal. So, there is no reason to introduce it here for new parameter.

It reminds me that this is approximately 27th place where is this pattern matching done. May be that we should think of some central processing of kernel parameters and generate e.g. a json for other componets.

Reply:
Yes, that's what I basically mentioned at the daily today. But I simply wanted to stay compatible with the other code for now. Till someone says let's remove it (and replace it with a shared functionality like a lib).

Reply:
well i thought that conclusion was, that we will keep it only for those params which already were implemented .

Anyway it is not blocker for me.